doc/source/cluster/kubernetes/examples/argocd.md

```diff
@@ -0,0 +1,461 @@
+(deploying-on-argocd-example)=
+
+# Deploying Ray Clusters via ArgoCD
+
+This guide provides a step-by-step approach for deploying Ray clusters on Kubernetes using ArgoCD. This example demonstrates how to deploy the KubeRay operator and a RayCluster with three different worker groups, leveraging ArgoCD's GitOps capabilities for automated cluster management.
+
+## Prerequisites
+
+Before proceeding with this guide, ensure you have the following:
+
+* A Kubernetes cluster with appropriate resources for running Ray workloads.
+* `kubectl` configured to access your Kubernetes cluster.
+* (Optional)[ArgoCD installed](https://argo-cd.readthedocs.io/en/stable/getting_started/) on your Kubernetes cluster.
+* (Optional)[ArgoCD CLI](https://argo-cd.readthedocs.io/en/stable/cli_installation/) installed on your local machine (recommended for easier application management).
```

Comment:
```suggestion
* (Optional)[ArgoCD CLI](https://argo-cd.readthedocs.io/en/stable/cli_installation/) installed on your local machine (recommended for easier application management. It might need [port-forwad and login](https://argo-cd.readthedocs.io/en/stable/getting_started/#port-forwarding) depending on the environment).
```